metadata.yaml

```diff
@@ -0,0 +1,104 @@
+---
+  id: "terraform-ibm-modules/atracker/ibm/1.4.0"
+  owner: "ibmtfprovider"
+  namespace: "terraform-ibm-modules"
+  name: "atracker"
+  version: "1.4.0"
+  providers: ["ibm"]
+  provider_logo_url: "/images/providers/generic.svg?2"
+  description : "Terraform module used to create activity tracker with storage solution on IBM Cloud "
+  source: "https://github.com/terraform-ibm-atracker"
+  owner: "ibmtfprovider"
+  tags: ["v1.4.0","activity-tracker"]
+  published_at: "2021-12-10T10:34:28.911362Z"
+  registry: ""https://registry.terraform.io/modules/terraform-ibm-modules/atracker/ibm/latest""
+  alias: "ob"
+###################################
+# Root module
+###################################
+root:
+  - path: ""
+    name: "atracker"
+    provider_dependencies:
+      - name: "ibm"
+        namespace: "IBM-Cloud"
+        source: "IBM-Cloud/ibm"
+        version: ""
+    inputs:
+      - name: "resource_group"
+        type : string
+        is_required: true
+        description: "Name of the resource group"
+        example: "default"
+        is_sensitive: false
+        related_to: ["data.ibm_resource_group.group"]
+        is_nullable: false
+        is_force_new: false
+        is_provision_controller: false
+        is_count_controller: false
+      - name: "bind_resource_key"
```

Comment:
Please capture variables information which are used at root module level in this section.

Sorry, I dont see a variable with name "bind_resource_key" at root level

please cross check the contents of metadata file once